dist/index.html

```diff
@@ -169,6 +169,24 @@
   bottom: 0;
   right: 0;
 }
+
+/* clouds styles */
+.sidebar.clouds {
+	overflow-y:scroll;
+	overflow-x:hidden;
+}
+/* hide unused parts */
+.sidebar.clouds .thingy .checkbox {
+  display: none;
+}
+.sidebar.clouds  .thingy.numeric-slider {
+    display: none;
+}
+.sidebar.clouds  fieldset label {
+    display: none;
+}
```

Comment:
Hmm, why are you hiding these?  I wanted to fix the layout of the sliders:

> ![image](https://user-images.githubusercontent.com/124687/121002654-907c9580-c75a-11eb-9cb0-0ebc276fdc00.png)

but when I used our `<NumericSlider>` component (the one in `numeric-slider.tsx`, not the one in `clouds.tsx`), it didn't show up, because this CSS was hiding it...

Reply:
Atul - thanks for your comments!  When I first started the Clouds page I just copied the Waves page. At that time the Numeric Slider definition was built in to the Waves page - so therefore it was in the Clouds page as well.  I now see you've modernized the Waves page - and the Numeric Slider is being included - so I'll do that on the Clouds page as well. Thanks for pointing that out!

Reply:
@toolness - I hid some controls in CSS because I included the Randomizer on clouds to randomize the colors - but the Randomizer widget \lib\randomizer-widget.tsx also included symbol controls - which don't apply to clouds (at least not yet) - so there were extraneous controls on the clouds Randomizer. So I used CSS to hide them - because I don't think there was another way to hide them.  One idea would be to add a parameter to the Randomizer Widget that hides the non-color controls.

Reply:
Ah I see, sure I can add options to the widget that allows hiding some of those options!